### PR TITLE
[MINOR][PAY-2015] update the docs regarding the new reimbursement apis

### PR DESCRIPTION
**Pull request by janos-fidel**
#### Steps to test

- Go to the website project.
- Run `env DOCS_BRANCH=this-pr-branch yarn docs` to download the docs files from this PR.
- Run `yarn start` and go to `/docs` to check the changes.

### CODE REVIEW

File: reimbursement.md [hunk 1]
```diff
@@ -220,12 +167,13 @@ Example `amount` set to `2.55` and custom `description` to `Earned Stars`.
 
 ```sh
 curl -X POST \
-  https://api.fidel.uk/v1/transactions/{transactionId}/reimbursement \
+  https://api.fidel.uk/v1/reimbursement \
```
Comment (PedroMGSilva) [inline reimbursement.md:170]: we want to keep the old structure here

File: reimbursement.md [hunk 2]
```diff
@@ -236,27 +184,18 @@ fileName:transaction.json
 {
```
Comment (PedroMGSilva) [inline reimbursement.md:184]: line 180 still refers to a reimbursement token

File: reimbursement.md [hunk 3]
```diff
@@ -220,44 +222,28 @@ Example `amount` set to `2.55` and custom `description` to `Earned Stars`.
 
 ```sh
 curl -X POST \
-  https://api.fidel.uk/v1/transactions/{transactionId}/reimbursement \
+  https://api.fidel.uk/v1/transactions/{transactionId}reimbursement \
```
Comment (pedroalmeida-fidel) [inline reimbursement.md:225]: I think we're missing a `/` 😁

File: reimbursement.md [hunk 4]
```diff
@@ -220,44 +222,28 @@ Example `amount` set to `2.55` and custom `description` to `Earned Stars`.
...
   -H 'Content-Type: application/json' \
   -H 'Fidel-Key: <KEY>' \
   -d '{
     "amount": 2.55,
-    "description": "Earned Stars"
+    "description": "Earned Stars",
```
Comment (LucasSindra) [inline reimbursement.md:230]: Any reason for this change?

File: reimbursement.md [hunk 5]
```diff
@@ -220,44 +222,28 @@ Example `amount` set to `2.55` and custom `description` to `Earned Stars`.
...
   }'
 ```
 
-After the reimbursement request is received by the card scheme, the full transaction is returned with the newly created `transaction.reimbursement` object with `pending` status, and also a `reimbursement.token` string, representing the unique identifier for that reimbursement request.
+After the reimbursement request is received by the card scheme, the full transaction is returned with the newly created `id`, representing the unique identifier for that reimbursement request.
```
Comment (LucasSindra) [inline reimbursement.md:234]: It says it returns the full transaction, but it should be only the id.

```suggestion
After the reimbursement request is received by the card scheme, the newly created `id` is returned, representing the unique identifier for that reimbursement request.
```

File: reimbursement.md [hunk 6]
```diff
@@ -271,13 +257,13 @@ Reimbursement `status` is set to `pending` while waiting for the card scheme to
 
 ### Status
 
-Find the reimbursement status in the transaction `reimbursement.status` property:
+Find the reimbursement status in the reimbursement's `status` property:
```
Comment (LucasSindra) [inline reimbursement.md:260]: Small one, but I'd say we can refactor this, now that it's much simpler to find.

```suggestion
The `status` property represents the possible reimbursement status as:
```

File: reimbursement.md [hunk 7]
```diff
@@ -295,72 +281,14 @@ fileName: request.json
 }
 ```
 
-The endpoint would then respond with data that would include card and reimbursement data, etc.
+The endpoint would then respond with unique identifier for the reimbursement that will be created for the most suitable transaction on the card.
```
Comment (LucasSindra) [inline reimbursement.md:284]:
```suggestion
The endpoint would then respond with the unique identifier for the reimbursement that will be created for the most suitable transaction on the card.
```

File: webhooks.md [hunk 8]
```diff
@@ -503,19 +503,21 @@ curl -X POST \
   }'
 ```
 
-`transaction.reimbursement.status` event is triggered when a reimbursement changes state on a transaction.
+`reimbursement.pending`, `reimbursement.issued`, `reimbursement.failed` events are triggered when a reimbursement changes state. The `reimbursement.pending` when the new state is `pending`, `reimbursement.issued` when a new state is `issued`, and `reimbursement.failed` when a new state is `failed`
```
Comment (LucasSindra) [inline webhooks.md:506]: Should we maybe document it like this?
"The `reimbursement.pending` when the reimbursement is created"
"and `reimbursement.failed` when it fails to complete the reimbursement"

Purely as example.

**Review by LucasSindra — CHANGES_REQUESTED** [review]

File: reimbursement.md (hunk 4 above)
Comment (PedroMGSilva) [reply reimbursement.md:230]: It can't have the comma here actually

File: reimbursement.md (hunk 3 above)
Comment (janos-fidel) [reply reimbursement.md]: thank you, good catch fixed here: https://github.com/FidelLimited/docs/pull/207/commits/44b7b13e5b6ec37498a3fe1b50d938b2730e570c

File: webhooks.md (hunk 8 above)
Comment (janos-fidel) [reply webhooks.md:506]: The lifecycle of the reimbursement is already explained in the document, and the webhook names are clearly mapped to statuses, it's clear enough in my opinion

**Review by pedroalmeida-fidel — APPROVED** [review]

**Review by LucasSindra — DISMISSED** [review]

**Review by PedroMGSilva — APPROVED** [review]

**Review by PedroMGSilva — DISMISSED** [review]

**Review by pedroalmeida-fidel — APPROVED** [review]

**Review by LucasSindra — APPROVED** [review]

**Review by LucasSindra — APPROVED** [review]

**Review by pedroalmeida-fidel — APPROVED** [review]

**Pull request merged**